### PR TITLE
mysql style timing and rows affected report for interpreter

### PR DESCRIPTION
**Pull request by xwb1989**

### CODE REVIEW

File: interpreter/main.go [hunk 1]
```diff
@@ -55,7 +56,9 @@ func saveHistory() {
 
 func executeLine(tx *sql.Tx, txnLine string) error {
 	if tidb.IsQuery(txnLine) {
+		start := time.Now()
```
Comment (ngaut) [inline interpreter/main.go]: Move up "start := time.Now()", there is no need to write it twice.

**Comment by ngaut** [comment]
Well done. Thanks @xwb1989

**Comment by siddontang** [comment]
LGTM

File: interpreter/main.go (hunk 1 above)
Comment (xwb1989) [inline interpreter/main.go]: My rationale was to make sure we only report the elapsed time of query execution, therefore the timing was put as close as possible to the query execution instructions; actually I do agree with you refactoring `start := time.Now()` is cleaner.

**Comment by shenli** [comment]
LGTM

**Pull request merged**